src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelInternal.java

```diff
@@ -58,6 +58,9 @@ class SnowflakeStreamingIngestChannelInternal<T> implements SnowflakeStreamingIn
   // Internal map of column name -> column properties
   private final Map<String, ColumnProperties> tableColumns;
 
+  // The cause of channel invalidation
+  private String invalidationCause;
```

Comment:
Could we make it clear in the comment that this is the latest cause that we see since a channel  could get invalidated multiple times?